packages/app-desktop/gui/TopBar/TopBar.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const ipcRenderer = require('electron').ipcRenderer;
+
+const minimizeHandler = () => {
+	ipcRenderer.send('minimize-handler');
+};
+const maximizeHandler = () => {
+	ipcRenderer.send('maximize-handler');
+};
+const closeWindowHandler = () => {
+	ipcRenderer.send('close-window-handler');
+};
+
+export default function TopBar() {
+	return (
+		<div>
+			Joplin
```

Comment:
The title isn't always Joplin. Also I think it might conflict with another open PR - https://github.com/laurent22/joplin/pull/4390